SpherePacking/Basic/PeriodicPacking.lean

```diff
@@ -871,7 +872,7 @@ theorem volume_ball_ratio_tendsto_nhds_one'
   · convert ENNReal.Tendsto.div (volume_ball_ratio_tendsto_nhds_one hd hC') ?_
       (volume_ball_ratio_tendsto_nhds_one hd hC) ?_ <;> simp
 
-theorem Filter.map_add_atTop_eq {β : Type*} {f : ℝ → β} (C : ℝ) (α : Filter β) :
+theorem Filter.map_add_atTop_eq' {β : Type*} {f : ℝ → β} (C : ℝ) (α : Filter β) :
```

Comment:
A conflicting lemma was added to mathlib in https://github.com/leanprover-community/mathlib4/pull/29532.